test/pensions/config/childrenAddress.unit.spec.jsx

```diff
@@ -34,11 +35,30 @@ describe('Child address page', () => {
     expect(formDOM.querySelectorAll('input, select, textarea').length).to.equal(2);
   });
 
+  it('should render address fields', () => {
+    const form = ReactTestUtils.renderIntoDocument(
+      <DefinitionTester
+          arrayPath={arrayPath}
+          pagePerItemIndex={0}
+          definitions={formConfig.defaultDefinitions}
+          schema={schema}
+          data={nameData}
+          uiSchema={uiSchema}/>
+    );
+
+    const formDOM = getFormDOM(form);
+    formDOM.selectRadio('root_childInHouseholdNo', 'N');
```

Comment:
Out of curiosity, is this the preferred way now? You could also do
```js
formDOM.selectYesNo('root_childInHousehold', 'N');
```
I wrote that one before `selectRadio`, though, so we could depreciate `selectYesNo` if we wanted.

Have you an opinion?